python-avocado.spec

```diff
@@ -45,7 +45,7 @@ BuildRequires: python3-devel
 BuildRequires: python3-docutils
 BuildRequires: python3-lxml
 BuildRequires: python3-psutil
-BuildRequires: python3-setuptools
+BuildRequires: python3-setuptools >= 40.6.3
```

Comment:
We have a problem here because this breaks RHEL/CentOS 8 builds.  Granted, CentOS 8 is going to be EOL'd in a couple of weeks, but this should be aligned with everything else RHEl/CentOS 8 related (including the newly added Cirrus CI tasks).